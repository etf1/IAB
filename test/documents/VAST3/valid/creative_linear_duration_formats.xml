<!-- <Creative>s with different <Duration> values -->
<VAST version="3.0">
    <Ad id="601364">
        <InLine>
            <AdSystem>Acudeo Compatible</AdSystem>
            <AdTitle>VAST 2.0 Instream Test 1</AdTitle>
            <Description>VAST 2.0 Instream Test 1</Description>
            <Error>http://myErrorURL/error</Error>

            <Impression>http://myTrackingURL/impression</Impression>
            <Creatives>
                <Creative AdID="601364">
                    <Linear>
                        <Duration>01:00:00</Duration>
                    </Linear>
                </Creative>
                <Creative AdID="601364">
                    <Linear>
                        <Duration>01:00:00.000</Duration>
                    </Linear>
                </Creative>
                <Creative AdID="601364">
                    <Linear>
                        <Duration>100:59:59.123</Duration>
                    </Linear>
                </Creative>
            </Creatives>
        </InLine>
    </Ad>
</VAST>
