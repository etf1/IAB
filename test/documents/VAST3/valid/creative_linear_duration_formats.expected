{
  "version": "3.0",
  "ads": [
    {
      "adType": 1,
      "adTitle": "VAST 2.0 Instream Test 1",
      "description": "VAST 2.0 Instream Test 1",
      "adSystem": {
        "name": "Acudeo Compatible"
      },
      "id": "601364",
      "error": "http://myErrorURL/error",
      "impressions": [
        {
          "uri": "http://myTrackingURL/impression"
        }
      ],
      "creatives": [
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adID": "601364"
        },
        {
          "creativeType": 1,
          "duration": "01:00:00.000",
          "adID": "601364"
        },
        {
          "creativeType": 1,
          "duration": "100:59:59.123",
          "adID": "601364"
        }
      ]
    }
  ]
}
